packages/@uppy/core/src/Uppy.js

```diff
@@ -1187,6 +1187,8 @@ class Uppy {
     }
     plugin.install()
 
+    this.emit('plugin-added', plugin)
```

Comment:
should we document this new event, or is it only part of the private API?

Reply:
We have `plugin-removed` and it’s currently undocumented, so this matches that. We could expose both, but not sure if there’s a common need for that?

Reply:
Can you write a comment (or update description) in #3938 so we track this name to be changed and whether it should be moved to an internal emitter?